### ISSUE TITLE
2.18 regression: errors out with empty SERVER_FALLBACK_PAGE

### ISSUE THREAD

**Issue by OdyX** [joseluisq/static-web-server#218]
**Describe the bug**
With the following Dockerfile, I could run the resulting image even with an empty `SERVER_FALLBACK_PAGE`:
```Dockerfile
ARG STATIC_WEB_SERVER_VERSION=2.17
FROM joseluisq/static-web-server:${STATIC_WEB_SERVER_VERSION}

# SWS configuration, check https://sws.joseluisq.net/configuration/environment-variables/
ENV SERVER_ROOT=/public

# Set fallback page for Single-page Applications
ARG SERVER_FALLBACK_PAGE
ENV SERVER_FALLBACK_PAGE=${SERVER_FALLBACK_PAGE}

# The whole website
COPY public/ /public/
```

Now, setting STATIC_WEB_SERVER_VERSION to 2.18, the same build gives:
```bash
error: a value is required for '--page-fallback <PAGE_FALLBACK>' but none was supplied
```
**To Reproduce**
Set an empty `SERVER_FALLBACK_PAGE` environment variable:

```bash
# Works
$ docker run -e SERVER_FALLBACK_PAGE= joseluisq/static-web-server:2.17
^C
# Fails
$ docker run -e SERVER_FALLBACK_PAGE= joseluisq/static-web-server:2.18
error: a value is required for '--page-fallback <PAGE_FALLBACK>' but none was supplied

For more information, try '--help'.
```

**Environment and Specs**
 - static-web-server: 2.18

This seems to be a regression caused by https://github.com/static-web-server/static-web-server/pull/215

**Comment by joseluisq** [comment]
Thanks for that!
We will fix that definitely, in fact, will be trivial to be fixed.
But for now, the "workaround" should be to unset the `SERVER_FALLBACK_PAGE` and avoid passing it to SWS if empty.

**Comment by OdyX** [comment]
In fact, I've found that I can pass `false` to it too;

```Dockerfile
ARG STATIC_WEB_SERVER_VERSION=2.18
FROM joseluisq/static-web-server:${STATIC_WEB_SERVER_VERSION}

# SWS configuration, check https://sws.joseluisq.net/configuration/environment-variables/
ENV SERVER_ROOT=/public

# Set fallback page for Single-page Applications
ARG SERVER_FALLBACK_PAGE=false
ENV SERVER_FALLBACK_PAGE=${SERVER_FALLBACK_PAGE}

# The whole website
COPY public/ /public/
```

This is thanks to https://docs.rs/clap/latest/clap/struct.Arg.html#method.env:

> Note that the value parser controls how flags are parsed. In this case we’ve selected [FalseyValueParser](https://docs.rs/clap/latest/clap/builder/struct.FalseyValueParser.html). A false literal is n, no, f, false, off or 0. An absent environment variable will also be considered as false. Anything else will considered as true.

**Comment by joseluisq** [comment]
> In fact, I've found that I can pass `false` to it too;

You can theoretically whatever string that [`Pathbuf`](https://doc.rust-lang.org/stable/std/path/struct.PathBuf.html) supports. But it will get validated when trying to read the fallback page content at server start-up time.

> This is thanks to https://docs.rs/clap/latest/clap/struct.Arg.html#method.env:
> 
> > Note that the value parser controls how flags are parsed. In this case we’ve selected [FalseyValueParser](https://docs.rs/clap/latest/clap/builder/struct.FalseyValueParser.html). A false literal is n, no, f, false, off or 0. An absent environment variable will also be considered as false. Anything else will considered as true.

Yeah, this is a rigid effect of our last migration to Clap v4.
Anyway, this is fixed now. See more details on PR #219.

**Comment by joseluisq** [comment]
You can use the `devel` Docker images to test the bugfix.

```
docker pull joseluisq/static-web-server:devel
docker pull joseluisq/static-web-server:devel-alpine
```

**Comment by joseluisq** [comment]
Fixed on [v2.19.0](https://github.com/static-web-server/static-web-server/releases/tag/v2.19.0)
